.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jquery": true
```

Comment:
Is that really necessary?

Reply:
I believe so. We're using browser objects, ECMAScript 6 features (such as `const`), and the `jQuery` object.

Reply:
janus.js doesn't use jQuery, only the demos do.

Reply:
To be fair, it'd make sense to extend the linting to all *.js files in a follow-up. Personally I'd leave this line out of this specific PR though since it seems to serve no purpose.

Reply:
I would 100% object linting the demos. The demos are not canon, they're examples: `janus.js` is the only thing that can end up in people's projects, and even then it's a sample implementation of the Janus API, and there are several (often better) JS stacks out there.

Reply:
janus.js refers to jQuery inside `Janus.useOldDependencies`.

Reply:
I completely understand if you have no interest in doing that yourself and such a follow-up may never even happen, but I find it surprising that you would actively object if someone's willing to put in the effort.

Reply:
What I meant is that I wouldn't want this in the project, so no point in anyone putting the effort. Whatever people contribute and gets merged, becomes something we need to maintain. This linting on janus.js will already be quite a burden on me (I'm not a JS developer), as any change I make I'll need to make sure it fits: while I understand why it may be needed for janus.js (which will be used in several projects) I see no point in doing the same for demos, where I want to be much more free to experiment and be "sloppy" since they're meant to be dumb example, and people should never base their products on those anyway.

Reply:
I guess this is not the right place to discuss the linting of js plugins.
Setting jquery globals in the linter does not sound like a big issue to me, so I'm happy to accept it.